Automattic/wp-calypso
# Onboarding Components: add README

## Changes proposed in this Pull Request

Pretty self-explanatory, added a breakdown of the props. I opted not to leave an example because we are using it already in IntentScreen and it would likely be overkill. I can add an example though if we think it is necessary. 

## Testing instructions

Read it and spot check

_Needs the package name update first, but then..._
Closes #61317
